zarr/creation.py

```diff
@@ -66,6 +67,9 @@ def create(shape, chunks=True, dtype=None, compressor='default',
         True if array should be protected against modification.
     object_codec : Codec, optional
         A codec to encode object arrays, only needed if dtype=object.
+    dimension_separator : {'.', '/'}, optional
+        Separator placed between the dimensions of a chunk.
+      .. versionadded:: 2.8
```

Comment:
```suggestion
        .. versionadded:: 2.8
```

Not sure about indentation of the directive.